src/coord/tests/minimal_resolution.rs

```diff
@@ -0,0 +1,104 @@
+// Copyright Materialize, Inc. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use std::error::Error;
+
+use tempfile::NamedTempFile;
+
+use coord::{
+    catalog::builtin::{MZ_CATALOG_SCHEMA, PG_CATALOG_SCHEMA},
+    session::Session,
+};
+use sql::{
+    catalog::Catalog,
+    names::{DatabaseSpecifier, FullName, PartialName},
+};
```

Comment:
Can you flatten these out per our usual import style? Btw, if these are a result of using rust-analyzer, you can configure the "last" mergeImports style to follow our style automatically.

Reply:
👍